api_tests/sanity/lnd.ts

```diff
@@ -0,0 +1,21 @@
+import { twoActorTest } from "../lib_sdk/actor_test";
+
+setTimeout(function() {
+    twoActorTest("sanity-lnd-alice-pays-bob", async function({ alice, bob }) {
+        await alice.startLnd();
+        await bob.startLnd();
+
+        await alice.fundLnd();
+
+        await alice.connectLnd(bob);
+        await alice.openChannel(bob);
+
+        const invoice = await bob.addInvoice(alice);
+        await alice.sendPayment(invoice);
+
+        await alice.assertChannelBalanceSender();
```

Comment:
I think it is better to (additionally) assert that the invoice was actually settled: e.g. 
```
alice.assertInvoiceSettled(invoice);
bob.assertInvoiceSettled(invoice);
```

Reply:
Right this second I don't see the benefit, I don't see any harm either.  I'll trust you on this one and add the above lines to the current test skeleton.  When I come to implement them I'll get a better understanding :)

Reply:
I think it is easier to verify that a invoice was settled than to verify the balance of a state. For the latter one you will need to remember the state at the beginning and at the end of the payment +- payment fees (which annoys me every time :D)

Reply:
Cool, thanks for saving me the time of have to work this out :)